test/org.jkiss.dbeaver.test.platform/src/org/jkiss/dbeaver/tools/transfer/DataExporterCSVTest.java

```diff
@@ -0,0 +1,66 @@
+import org.jkiss.dbeaver.model.data.DBDAttributeBinding;
```

Comment:
Please add copyright as it's done in other files.

Reply:
Also package declaration is missing

Reply:
I have added these in the latest commit.